src/insns/modesw_common.adoc

```diff
@@ -2,8 +2,7 @@
 Description::
 Toggle the hart's current CHERI execution mode in <<pcc>>.
 +
-* If the current mode in <<pcc>> is pass:attributes,quotes[{cheri_int_mode_name}], then the mode bit (M) in <<pcc>> is set to pass:attributes,quotes[{cheri_cap_mode_name}].
-* If the current mode is pass:attributes,quotes[{cheri_cap_mode_name}], then the mode bit (M) in <<pcc>> is set to pass:attributes,quotes[{cheri_int_mode_name}].
+* If the current mode in <<pcc>> is pass:attributes,quotes[{cheri_int_mode_name}] ({INT_MODE_VALUE}), then the <<m_bit>> in <<pcc>> is set to pass:attributes,quotes[{cheri_cap_mode_name}] ({CAP_MODE_VALUE}).
+* If the current mode is pass:attributes,quotes[{cheri_cap_mode_name}] ({CAP_MODE_VALUE}), then the <<m_bit>> in <<pcc>> is set to pass:attributes,quotes[{cheri_int_mode_name}] ({INT_MODE_VALUE}).
 +
-In debug mode MODESW can still be used to change the operating mode, and the current
-mode is shown in the <<m_bit>> of <<dinfc>>.
+The current mode can be observed as shown in <<m_bit_observe>>.
```

Comment:
Please note that the _current mode_ and the value of PCC's _m-bit_ are related, but they are not the same. The section linked to in this sentence is for discovering the current mode, not the PCC's _m-bit_ which is what the MODESW instruction covers.